odd-platform-ui/src/components/DataEntityDetails/DatasetStructure/DatasetStructureTable/DatasetStructureList/DatasetStructureList.tsx

```diff
@@ -78,6 +78,7 @@ const DatasetStructureList: React.FC<DatasetStructureListProps> = ({
             rowCount={datasetRowsCount}
             rowHeight={cache.rowHeight}
             rowRenderer={renderListItem}
+            scrollToIndex={scrollToIndex}
```

Comment:
Also, i thin it will be better, if you pass `scrollToAlignment` prop to `start`, in this case align of scrolled row will be at the top of the list.